test/graphql-api/compatibility.node.test.ts

```diff
@@ -1,6 +1,5 @@
 import fetch from 'cross-fetch'
-import { graphql as executeGraphql } from 'graphql'
-import { buildSchema } from 'graphql/utilities'
```

Comment:
we use this from `/utilities` and from `graphql` in different files, so I moved to just using the base package export

Reply:
💯 agree with this. We should rely on the base package all the time.